StripeIdentity/StripeIdentity/Source/NativeComponents/ViewControllers/DocumentCaptureViewController.swift

```diff
@@ -367,10 +375,12 @@ final class DocumentCaptureViewController: IdentityFlowViewController {
     private func saveFrontAndDecideBack(
         frontImage: UIImage
     ) {
+        isDecidingBack = true
         sheetController?.saveDocumentFrontAndDecideBack(
             from: analyticsScreenName,
             documentUploader: documentUploader,
             onCompletion: { [weak self] isBackRequired in
+                self?.isDecidingBack = false
```

Comment:
I'm just flagging this just-in-case. I don't know if this is actually any issue.

Here we set `self?.isDecidingBack = false`, saying that BACK is not decided. 

There's code below that would make it sound like it will try to decide to scan back:

```
                if isBackRequired {
                    self?.imageScanningSession.startScanning(
                        expectedClassification: DocumentSide.back
                    )
                    self?.updateUI()
                } 
```

The worry is that this "back" code is wrong considering the `decidingBack = false` state. 

I might be completely wrong on calling it out, just flagging.

Reply:
ah actually it doesn't mean "back is not decided", it means "back is not _being_ decided", this boolean would toggle the button to loading state and block any other button click while the logic is running. After back is decided we'll need to reenable the button